src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -743,11 +838,59 @@ public NoteModel AddNote(bool centerNote, double xPos, double yPos, string text,
             return noteModel;
         }
 
+        public void ClearNotes()
+        {
+            lock (notes)
+            {
+                notes.Clear();
+            }
+
+            OnNotesCleared();
+        }
+
+        private void RemoveNote(NoteModel note)
+        {
+            lock (notes)
+            {
+                if (!notes.Remove(note)) return;
+            }
+            OnNoteRemoved(note);
+        }
+
+        private void AddNewAnnotation(AnnotationModel annotation)
```

Comment:
I wanted to name it like
```c#
private void AddAnnotation(AnnotationModel annotation)
```
but there is public method with the same name and the same incoming parameter. (line 889)